packages/oasgraph/src/oas_3_tools.ts

```diff
@@ -1036,4 +1036,11 @@ export function capitalize(str: string): string {
  */
 export function uncapitalize(str: string): string {
   return str.charAt(0).toLowerCase() + str.slice(1)
+}
+
+/**
+ * For operations that do not have an operationId, generate one
+ */
+export function generateOperationId(method: string, path: string): string {
+  return ((beautify(`${method}:${path}`) as any) as string)
```

Comment:
Strange type conversion. Is that really necessary?

Reply:
Hmmm, you're right... I refactored old code and I didn't think much of it afterwards. I'll change it.